packages/vdom/vdom.0.3/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "DOM and VDOM for OCaml"
+description: """
+This package contains:
+- OCaml bindings to DOM and other client-side Javascript APIs (using gen_js_api).
+- An implementation of the Elm architecture, where the UI is specified as a functional "view" on the current state."""
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "Marc Lasson <marc.lasson@lexifi.com>"
+  "Aurélien Saue <aurelien.saue@lexifi.com>"
+]
+homepage: "https://github.com/LexiFi/ocaml-vdom"
+license: "MIT"
+bug-reports: "https://github.com/LexiFi/ocaml-vdom/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "gen_js_api" {>= "1.0.6"}
+  "js_of_ocaml-compiler"
```

Comment:
```suggestion
  "ojs"
  "js_of_ocaml-compiler"
```
due to
```

#=== ERROR while compiling vdom.0.3 ===========================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.10.2 | pinned(https://github.com/LexiFi/ocaml-vdom/archive/refs/tags/0.3.tar.gz)
# path                 ~/.opam/4.10/.opam-switch/build/vdom.0.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p vdom -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/vdom-7-a5523d.env
# output-file          ~/.opam/log/vdom-7-a5523d.out
### output ###
# File "lib/dune", line 17, characters 12-15:
# 17 |  (libraries ojs vdom_base)
#                  ^^^
# Error: Library "ojs" not found.
# -> required by library "vdom" in _build/default/lib
# -> required by _build/default/META.vdom
# -> required by _build/install/default/lib/vdom/META
# -> required by _build/default/vdom.install
# -> required by alias install
```

Reply:
I think it is better to require gen_js_api >= 1.0.7

Reply:
Feel free to update the bound, but the package explicitly links against `ojs` so it should be there as a dependency